main.go

```diff
@@ -195,7 +195,13 @@ func main() {
 		os.Exit(0)
 	}
 
-	state, err := command.GetGlobalState(globalArgs)
+	var state *command.GlobalState
```

Comment:
I believe it is possible to remove this if and call command.GetGlobalState directly like:
command.GetGlobalState(globalArgs, globalArgs.Command != "init")